DXMainClient/DXGUI/Multiplayer/GameLobby/CnCNetGameLobby.cs

```diff
@@ -1480,7 +1482,12 @@ protected override void BroadcastDiceRoll(int dieSides, int[] results)
             channel.SendCTCPMessage($"{DICE_ROLL_MESSAGE} {dieSides},{resultString}", QueuedMessageType.CHAT_MESSAGE, 0);
             PrintDiceRollResult(ProgramConstants.PLAYERNAME, dieSides, results);
         }
+        protected override void BroadcastHandRPS(string results)
+        {
 
+            channel.SendCTCPMessage($"{Hand_RPS_MESSAGE} {results}", QueuedMessageType.CHAT_MESSAGE, 0);
```

Comment:
imagine someone using a different language

Reply:
Also you shouldn't broadcast the display string. Broadcast only necessary raw parameters